src/webui/api/torrentscontroller.cpp

```diff
@@ -2143,3 +2143,20 @@ void TorrentsController::onMetadataDownloaded(const BitTorrent::TorrentInfo &inf
             iter.value().setTorrentInfo(info);
     }
 }
+
+void TorrentsController::setCommentAction()
+{
+    requireParams({u"hash"_s, u"comment"_s});
+
+    const auto id = BitTorrent::TorrentID::fromString(params()[u"hash"_s]);
+
+    BitTorrent::Torrent *const torrent = BitTorrent::Session::instance()->getTorrent(id);
+    if (!torrent)
+        throw APIError(APIErrorType::NotFound);
+
+    QString comment = params()[u"comment"_s].trimmed();
+    comment.replace(QRegularExpression(u"\r?\n|\r"_s), u" "_s);
+    torrent->setComment(comment);
+
+    setResult(QString());
```

Comment:
You don't actually need to do this.

Reply:
you mean specifically this part? `.replace(QRegularExpression(u"\r?\n|\r"_s), u" "_s);`

Reply:
> you mean specifically this part? `.replace(QRegularExpression(u"\r?\n|\r"_s), u" "_s);`

I mean exactly the line I commented on: `setResult(QString());`.